demo/DemoArticles/update.js

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env node
+/* eslint-disable require-jsdoc */
+
+const fs = require('fs')
+const request = require('request')
+const indexJSON = require('./articles.json')
+const ArticleRef = require('./ArticleRef.js').ArticleRef
+const ArticleRefSourceType = require('./ArticleRef.js').ArticleRefSourceType
+const flattenArrayOfArrays = require('../DemoUtilities.js').flattenArrayOfArrays
+
+const DATA_PATH = './data/'
+
+const articleRefs = flattenArrayOfArrays(
+  indexJSON.map(articleData => [
+    new ArticleRef(
+      articleData.lang,
+      articleData.title,
+      articleData.revision,
+      ArticleRefSourceType.mobileView
+    ),
+    new ArticleRef(
+      articleData.lang,
+      articleData.title,
+      articleData.revision,
+      ArticleRefSourceType.mobileContentService
+    )
+  ])
+)
+
+// eslint-disable-next-line no-console
+console.log(`Fetching JSON for ${articleRefs.length} titles...`)
+
+// Linting complained of some improperly formatted characters which turned out to be these.
```

Comment:
Did you see my link? json.org specifies that a JSON string is composed of chars. chars are:
- any-Unicode-character-except-"-or-\-or-control-character
- \"
- \\
- \/
- \b
- \f
- \n
- \r
- \t
- \u four-hex-digits

This code is just escaping it it in the dumbest way possible. We just need to generically map all illegal characters to escaped versions of those characters and decide whether to escape them as Unicode characters (current implementation) or HTML entities.

It could also just be that our JSON linter isn't working right but I think the first order of investigation would be to peak at the linter source (especially since it's likely written in JavaScript).